pkg/app/bootstrap.go

```diff
@@ -146,7 +176,12 @@ func registryCA(clicontext *cli.Context) error {
 		if !exist {
 			return errors.Newf("file %s doesn't exist", kv[1])
 		}
-		path, err := fileutil.ConfigFile(fmt.Sprintf("registry_%s.pem", kv[0]))
+
+		if len(registry) == 0 {
+			registry = "docker.io"
+		}
+
+		path, err := fileutil.ConfigFile(fmt.Sprintf("%s_%s.pem", registry, kv[0]))
```

Comment:
Will it be better to set `docker.io` as the default value for the argument `registry`?

Reply:
I set it to `docker.io` because that's what the original config template set it to. Is there a better default? 

In practice however, I assume a user will use both `registry` and `registry-ca-keypair` together, otherwise you're providing credentials to an unknown registry. Maybe we should add a check to ensure they use both?

Reply:
`registry-ca-keypair` should require the `registry`. I mean you can set a default value like https://github.com/tensorchord/envd/blob/5383e01882204da67a68eff856110807b31130f5/pkg/app/bootstrap.go#L51. So you don't need to check if it's an empty string.